descriptors/coreboot-qemu-centosX100.yaml

```diff
@@ -0,0 +1,47 @@
+JobName: Qemu Boot CentOS 
+Runs: 100
+RunInterval: 60s
+Version: "1.0"
+Testdescriptors:
+    -   TargetManagerName: TargetList
+        TargetManagerAcquireParameters:
+          Targets:
+              - ID: "x86_64"
+          MinNumberDevices: 1
+          MaxNumberDevices: 1
+        TargetManagerReleaseParameters:
+        TestFetcherName: literal
+        TestFetcherFetchParameters:
+            TestName: Boot CentOS coreboot
+            Steps:
+              - name: qemu
+                label: qbt 
```

Comment:
We should change this to something like "Qemu Boot Test"

Reply:
We already did in the other PR. Lets set the wording first and than add new tests.